README.md

```diff
@@ -1,5 +1,5 @@
-# al# alpha-ec-be
-[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/alpha-ec-be/badge.svg)](https://coveralls.io/github/atlp-rwanda/alpha-ec-be)
+
+[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/alpha-ec-be/badge.svg?branch=develop)](https://coveralls.io/github/atlp-rwanda/alpha-ec-be?branch=develop)
```

Comment:
here use this 
`[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/alpha-ec-be/badge.svg)](https://coveralls.io/github/atlp-rwanda/alpha-ec-be)`

 insteady